src/Utilities/constants.js

```diff
@@ -25,6 +30,8 @@ export const INVENTORY_TOTAL_FETCH_EDGE_PARAMS =
 export const INVENTORY_TOTAL_FETCH_CONVENTIONAL_PARAMS = '?page=1&per_page=1';
 export const INVENTORY_TOTAL_FETCH_BIFROST_PARAMS =
   '?filter[system_profile][bootc_status][booted][image_digest][is]=not_nil&per_page=1';
+export const INVENTORY_FETCH_BIFROST_PARAMS =
```

Comment:
If you feel up to small improvements, we can combine the above 2 constants together. This would be useful in case backend folks change the filter syntax.